generate-stackbrew-library.sh

```diff
@@ -25,6 +25,22 @@ dockerfileCommit() {
 	)
 }
 
+getArches() {
+	local repo="$1"; shift
+	local officialImagesUrl='https://github.com/docker-library/official-images/raw/master/library/'
+
+	eval "declare -g -A parentRepoToArches=( $(
+		find -name 'Dockerfile' -exec awk '
+				toupper($1) == "FROM" && $2 !~ /^('"$repo"'|scratch|microsoft\/[^:]+)(:|$)/ {
+					print "'"$officialImagesUrl"'" $2
+				}
+			' '{}' + \
+			| sort -u \
+			| xargs /home/admin/prog/docker/official-images/bashbrew/go/bin/bashbrew cat --format '[{{ .RepoName }}:{{ .TagName }}]="{{ join " " .TagEntry.Architectures }}"'
```

Comment:
Is this the `bashbrew` path you intended to include in this PR?

Reply:
Sorry, forgot to prune that.

Reply:
@tianon would you like to review the updated PR?